### PR TITLE
fix(securityEvents): Tweak securityEvents db queries based on @jrgm feedback

### PR DESCRIPTION
**Pull request by rfk**
Here's some proposed tweaks to the DB queries for security events, based on feedback/concerns from @jrgm in https://bugzilla.mozilla.org/show_bug.cgi?id=1304287#c8

* Add `createdAt` to the index we use for fetching the events
* Store the tokenVerificationId on the event, so that we can directly look it up to verify it rather than indirecting through the tokenId.

The fetch query no longer threatens to use a filesort:

```
mysql> explain     SELECT         n.name,         e.verified,         e.createdAt     FROM         securityEvents e     LEFT JOIN securityEventNames n         ON e.nameId = n.id     WHERE         e.uid = unhex('000000111')     AND         e.ipAddr
+----+-------------+-------+------------+--------+-----------------------------------------------------------------------------------------------------------+-----------------------------------------+---------+--------------+------+----------+-------------+
| id | select_type | table | partitions | type   | possible_keys                                                                                             | key                                     | key_len | ref          | rows | filtered | Extra       |
+----+-------------+-------+------------+--------+-----------------------------------------------------------------------------------------------------------+-----------------------------------------+---------+--------------+------+----------+-------------+
|  1 | SIMPLE      | e     | NULL       | ref    | securityEvents_uid_tokenId,securityEvents_uid_tokenVerificationId,securityEvents_uid_ipAddrHmac_createdAt | securityEvents_uid_ipAddrHmac_createdAt | 48      | const,const  |    1 |    10.00 | Using where |
|  1 | SIMPLE      | n     | NULL       | eq_ref | PRIMARY                                                                                                   | PRIMARY                                 | 4       | fxa.e.nameId |    1 |   100.00 | NULL        |
+----+-------------+-------+------------+--------+-----------------------------------------------------------------------------------------------------------+-----------------------------------------+---------+--------------+------+----------+-------------+
```

@seanmonstar r?

### CODE REVIEW

**Comment by rfk** [comment]
Actually looks like this is currently a bit busted, taking it back to "active" for now...

**Comment by rfk** [comment]
OK, this version is looking more sensible :-)

File: lib/db/schema/patch-035-036.sql [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+--
+--  2) Use an index on (uid, ipAddrHmac, createdAt) to replace the
+--     one on (uid, ipAddrHmac), since we want to sort the results
+--     anyway.
+--
+-- It's a little complicated in order to support forward and backward
+-- compatibility, and it makes an important assumtion:
+--
+--  NB:  we assume that the previous deployment has security events
+--       disabled, and hence that there's no data already in the table.
```
Comment (seanmonstar) [inline lib/db/schema/patch-035-036.sql]: Would like to have @jrgm agree that this is the case.

Comment (rfk) [inline lib/db/schema/patch-035-036.sql]: assigning @jrgm for additional confirmation

Comment (jrgm) [inline lib/db/schema/patch-035-036.sql]: Confirmed.  `select * from securityEvent` => `Empty set (0.00 sec)`

**Pull request merged**